sdcflows/workflows/fit/syn.py

```diff
@@ -583,25 +594,52 @@ def _remove_first_mask(in_file):
     return workflow
 
 
-def _warp_dir(fixed_image, pe_dir, nlevels=3):
+def _warp_dir(moving_image, fixed_image, pe_dir, nlevels=3):
```

Comment:
We have 2 levels of registration only (I suspect that's because the affine component was removed into the preprocessing):

```suggestion
def _warp_dir(moving_image, fixed_image, pe_dir, nlevels=2):
```